# HG changeset patch
# User John W. Eaton <jwe@octave.org>
# Date 1310852869 14400
#      Sat Jul 16 17:47:49 2011 -0400
# Node ID 4dfc13ca35b94a0764dcca7737980a80af680d16
# Parent  e9b4de878a9cf616f746ff22b7742913b3e11a56
codesprint: sprinting too fast

* now.m: Fix one typo and one thinko in previous change.

diff --git a/scripts/time/now.m b/scripts/time/now.m
--- a/scripts/time/now.m
+++ b/scripts/time/now.m
@@ -51,11 +51,11 @@ function t = now ()
   ##
   ## mktime(gmtime(0)) does indeed return the offset from Greenwich to the
   ## local time zone, but we need to account for daylight savings time
   ## changing by an hour the offset from CUT for part of the year.
 
 endfunction
 
 %!error now (1);
-%!assert (isnumeric (now ());
+%!assert (isnumeric (now ()));
 %!assert (now () > 0);
-%!assert (now () >= now ());
+%!assert (now () <= now ());
